### PR TITLE
i18n(zh-cn): Create missing-locale.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Create missing-locale.mdx

#### Related issues & labels (optional)

- #6253

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/missing-locale.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: 提供的语言环境不存在。
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/missing-locale.mdx:6]: delete

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/reference/errors/missing-locale.mdx (hunk 1 above)
Comment (huyikai) [reply src/content/docs/zh-cn/reference/errors/missing-locale.mdx]: Completed.

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks y'all!

**Pull request merged**